app/src/main/java/com/dimagi/biometric/fragments/FaceMatchFragment.java

```diff
@@ -100,16 +100,41 @@ public void onTimedout(byte[] faceImage) {
     protected ParamManager getParams() {
         Intent intent = requireActivity().getIntent();
         ParamManager params = new ParamManager();
-        params.setPitch(intent.getIntExtra(ParamConstants.PITCH_NAME, ParamConstants.PITCH_DEFAULT));
-        params.setYaw(intent.getIntExtra(ParamConstants.YAW_NAME, ParamConstants.YAW_DEFAULT));
-        params.setRoll(intent.getIntExtra(ParamConstants.ROLL_NAME, ParamConstants.ROLL_DEFAULT));
-        params.setMask(intent.getFloatExtra(ParamConstants.MASK_NAME, ParamConstants.MASK_DEFAULT));
-        params.setSunglasses(intent.getFloatExtra(ParamConstants.SUNGLASSES_NAME, ParamConstants.SUNGLASSES_DEFAULT));
-        params.setEyesClosed(intent.getFloatExtra(ParamConstants.EYES_CLOSED_NAME, ParamConstants.EYES_CLOSED_DEFAULT));
-        params.setBrisque(intent.getIntExtra(ParamConstants.BRISQUE_NAME, ParamConstants.BRISQUE_DEFAULT));
-        params.setImageCenterTolerance(intent.getFloatExtra(ParamConstants.IMAGE_CENTER_TOLERANCE_NAME, ParamConstants.IMAGE_CENTER_TOLERANCE_DEFAULT));
-        params.setAutoCaptureEnabled(intent.getBooleanExtra(ParamConstants.AUTO_CAPTURE_ENABLED_NAME, ParamConstants.AUTO_CAPTURE_ENABLED_DEFAULT));
-        params.setTimeoutSecs(intent.getIntExtra(ParamConstants.TIMEOUT_SECS_NAME, ParamConstants.TIMEOUT_SECS_DEFAULT));
+
+        params.setPitch(safeParseInteger(
+                intent.getStringExtra(ParamConstants.PITCH_NAME), ParamConstants.PITCH_DEFAULT
+        ));
+        params.setYaw(safeParseInteger(
+                intent.getStringExtra(ParamConstants.YAW_NAME), ParamConstants.YAW_DEFAULT
+        ));
+        params.setRoll(safeParseInteger(
+                intent.getStringExtra(ParamConstants.ROLL_NAME), ParamConstants.ROLL_DEFAULT
+        ));
+        params.setMask(safeParseFloat(
+            intent.getStringExtra(ParamConstants.MASK_NAME),
+            ParamConstants.MASK_DEFAULT
+        ));
```

Comment:
Nit: Formatting seems off

Reply:
The CommCare inspection profile isn't reporting any issues with this code formatting. Do you have a suggestion or example on how this should be formatted?

Reply:
Try `Code` -> `Reformat Code`

Reply:
That did the trick ([2a0521f](https://github.com/dimagi/biometric-android/pull/10/commits/2a0521f0bcb9b434e079a514d26c34fb3b0fe737)).